### PR TITLE
card component added

### PR DESCRIPTION
**Pull request by Sharmaryan**
Screenshot (16)
Screenshot (17)
Screenshot (18)
Screenshot (19)

**card done, not responsive**

### CODE REVIEW

File: styles/card/card.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```
Comment (saadahmadhere) [inline styles/card/card.css:5]: consider giving line space after each {}

File: styles/card/card.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+.card-title {
+  margin: .4rem 0;
+  font-size: 1.4rem;
+  font-weight: bold;
+}
+.card-btn {
+  text-transform: uppercase;
+  font-weight: bold;
+  padding: 0.4rem;
```
Comment (saadahmadhere) [inline styles/card/card.css]: for value less than 10px, use pixels

File: styles/card/card.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  margin: .1rem;
+  cursor: pointer;
+}
+.card-logo {
+  height: 150px;
+  width: 200px;
```
Comment (saadahmadhere) [inline styles/card/card.css]: use rems for dynamic units

File: styles/card/card.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+.card-title {
+  margin: .4rem 0;
+  font-size: 1.4rem;
+  font-weight: bold;
+}
+.card-btn {
+  text-transform: uppercase;
+  font-weight: bold;
+  padding: 0.4rem;
+  margin: .1rem;
```
Comment (saadahmadhere) [inline styles/card/card.css]: 1.6px or 2px would be more suited here that 0.1rem

**Review by saadahmadhere — CHANGES_REQUESTED** [review]
Nice use of variables. Some space and appropriate use of units would make the code better.

File: styles/card/card.css [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+}
+.card-vertical{
+  background-color: white;
+  color: var(--secondary-color);
+  border: 1px solid var(--secondary-color);
+  width: 17rem;
+}
+.card-btns{
+  display: flex;
+
```
Comment (rishbitsnbytes) [inline styles/card/card.css]: Remove spacing

File: styles/card/card.css [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+}
+.card-horizontal{
+  display: flex;
+  width: 30rem;
+  background-color: var(--primary-color);
+  color: var(--secondary-color);
+  border: 1px solid var(--secondary-color);
+}
+.card-horizontal-logo{
+margin: auto 0;
```
Comment (rishbitsnbytes) [inline styles/card/card.css:77]: Use prettier for proper formatting.

**Review by rishbitsnbytes — COMMENTED** [review]
Minor comments, rest looks good.

File: styles/card/card.css (hunk 1 above)
Comment (Sharmaryan) [reply styles/card/card.css:5]: done

File: styles/card/card.css (hunk 2 above)
Comment (Sharmaryan) [reply styles/card/card.css]: done

File: styles/card/card.css (hunk 3 above)
Comment (Sharmaryan) [reply styles/card/card.css]: done

File: styles/card/card.css (hunk 4 above)
Comment (Sharmaryan) [reply styles/card/card.css]: done

File: styles/card/card.css [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  font-weight: bold;
+  font-size: 1.2rem;
+}
+
+.card-desc{
+  margin: 1rem 0;
+}
+
+.card-vertical{
+  background-color: white;
```
Comment (AtulHande03a) [inline styles/card/card.css]: Use variables for colors,

File: styles/card/card.css (hunk 5 above)
Comment (Sharmaryan) [reply styles/card/card.css]: done

**Review by Himanshu-Dhawale — APPROVED** [review]

File: styles/card/card.html [hunk 8]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Card | gotham UI</title>
```
Comment (AtulHande03a) [inline styles/card/card.html:7]: Add meta tags for author and description.

**Review by AtulHande03a — APPROVED** [review]
Apart from the comments rest all looks good!!

File: styles/card/card.html (hunk 8 above)
Comment (Sharmaryan) [reply styles/card/card.html:7]: when i will do refactoring , i will change in all of the components

File: styles/card/card.css (hunk 7 above)
Comment (Sharmaryan) [reply styles/card/card.css]: sure

Comment (Sharmaryan) [reply styles/card/card.css]: will refactor with comments in all components

**Review by saadahmadhere — APPROVED** [review]
Seems beautiful. Good to merge

**Pull request merged**